config/locales/rs.yml

```diff
@@ -515,29 +515,29 @@ rs:
       our_events: Naši događaji
       our_partners: Naši partneri
     about:
-      description: S95 je otvoren platform koji objedinjuje nezavisne park trke, koje se održavaju prema tri postulata - subota, 9 ujutru, 5 km.
+      description: S95 je otvorena platforma koja objedinjuje nezavisne park trke koje se održavaju prema tri postulata - subota, 9 ujutru, 5 km.
       events: 'Sistem trenutno prikazuje rezultate %{events_count} lokacija: %{events}.'
       join_us:
-        title: Želite da se pridružite platform S95?
+        title: Želite da se pridružite platformi S95?
         body_html: >
-          <p>In short, there is needed your desire.</p>
+          <p>Ukratko, potrebna je samo volja.</p>
           <p>
-            In more detail, you need:
+            Detaljnije, potrebni su:
             <ul>
-              <li>safe and comfortable route</li>
-              <li>a team that ready to handle runs every saturday all year round</li>
-              <li>a desire to make everyone healthier and happier</li>
+              <li>bezbedna i pristupačna staza</li>
+              <li>tim koji je voljan da organizuje trke svake subote tokom cele godine</li>
+              <li>zelja da se pomogne svima da budu zdraviji i srećniji</li>
             </ul>
           </p>
-          <p>Are you ready? Pišite nam na <a href="mailto:%{info_email}" target="_blank" title="Pošaljite nam email">%{info_email}</a> and soon jogging will appear in your park.</p>
+          <p>Da li ste spremni? Pišite nam na <a href="mailto:%{info_email}" target="_blank" title="Pošaljite nam email">%{info_email}</a> i uskoro će se trčati i u vašem parku.</p>
       s95_abroad:
-        title: S95 races in other countries
-        body: Today our run events are held also in %{countries}. All results are in one system saved and you can see it in athlete profiles.
+        title: S95 trke u drugim zemljama
+        body: Danas se naše trke organizuju i u %{countries}. Svi rezultati se čuvaju u jednom sistemu odakle ćete imati uvid u profile atletiča.
       back: Nazad
       forward: Napred
     rules:
       description: Brief rules for S95 participants, as well as frequently asked questions
-      register_with_bot_html: 'If you are not registered, create a profile using <a href="%{link}" target="_blank" rel="noopener">Telegram bot</a>. You will receive a link to your profile with a personal qr-code.'
+      register_with_bot_html: 'If you are not registered, create a profile using <a href='%{link}' target='_blank' rel='noopener'>Telegram bot</a>. You will receive a link to your profile with a personal qr-code.'
```

Comment:
Please, leave here double quotes, there is a parsing error

Reply:
It seems that VSCode's auto-format kicked in without me noticing :(